cmd/pbstatuser/main.go

```diff
@@ -64,6 +64,7 @@ func processMessage(origCtx context.Context, message *kafka.GenericMessage) {
 		logger.Warn().Err(err).Msgf("could not get availability status message %s", err)
 		return
 	}
+	logger.Trace().Msgf("received a message from sources to be processed with source id %s", asm.SourceID)
```

Comment:
I know this will be pesky but:

* errors all lowercase
* logs start uppercase

If only I could change this and keep logs lowercase too.

```suggestion
	logger.Trace().Msgf("Received a message from sources to be processed with source id %s", asm.SourceID)
```